### DISCUSSION TITLE
.mjmlignore to ignore some linting rules

### DISCUSSION THREAD

**Discussion by Ostefanini** [mjmlio/mjml#2512] [Ideas]
Hello
Awesome package here !
However, to perform compatibility accross client software emails, we have to "disgress" few linting rules. We know that theses rules are here for a good reason, however our internal tests shows good results.

We plan to do a refactorisation in the future, but in the meantime, we we'd like to use `mjml --validate` feature in our CI. Currently, we can't  because of those rules transgression.

Do you think it might be possible to add a mechanism of ignoring some linting rules, like .eslintignore ?

Thank you for the consideration

**Comment by iRyusa** [comment]
I think that would be great to add as well as other validator rules to register
